### PR TITLE
[defns.erroneous], [basic.indet] Index 'erroneous behavior' and 'erroneous value'

### LINKED ISSUE #7043: [defns.erroneous] Erroneous behavior is not indexed in `/generalindex`
[defns.undefined] indexes "undefined behavior"/"behavior, undefined" as follows:
https://github.com/cplusplus/draft/blob/4b3f32ae814c8da3faccc0dc307904bd250371d9/source/intro.tex#L629-L632

[defns.erroneous] only contains a `\definition`:
https://github.com/cplusplus/draft/blob/4b3f32ae814c8da3faccc0dc307904bd250371d9/source/intro.tex#L251-L252

### PR DESCRIPTION
**Pull request by Eisenwave**
Fixes #7043.

### CODE REVIEW

File: source/basic.tex [hunk 1]
```diff
@@ -3750,6 +3750,8 @@
 \pnum
 \indextext{value!indeterminate}%
 \indextext{indeterminate value}%
+\indextext{value!erroneous}%
```
Comment (jensmaurer) [inline source/basic.tex:3753]: I'm wondering whether we should make "erroneous value" a defined term and simply use \defnadjx.  That feels a lot more straightforward.

Comment (Eisenwave) [reply source/basic.tex:3753]: I could do that too. If we're going to do that, I think we should make the same change for indeterminate values.

That's also why I've just perpetuated whatever is there right now; perhaps the conversion of both terms to `\denfadjx` could be done in a follow-up PR.

Comment (Eisenwave) [reply source/basic.tex:3753]: Alright, so I've noticed that "indeterminate value" has both the `\defnadjx` definition and the `\indextext` up there.

Now, it's the same for erroneous values. Is there a reason why indeterminate values have both? Can/should we just get rid of all those `\indextext` entries?

Comment (Eisenwave) [reply source/basic.tex:3753]: When removing `\indextext`, all index entries still exist, but we lose the link within the PDF and the page number.
image

With `\indextext`, there's a `68` at the end.

Comment (jensmaurer) [reply source/basic.tex:3753]: Yup, that's intentional: we want the "see" entry, not a direct page number.

Comment (jensmaurer) [reply source/basic.tex:3753]: So, please remove \indextext and use \defnadjx

**Review by jensmaurer — APPROVED** [review]

**Pull request merged**